file_aloc/bl_import_scenario.py

```
@@ -0,0 +1,195 @@
+import bpy
+import bmesh
+from .format import PhysicsCollisionLayerType, PhysicsCollisionType
+
+
+def log(level, msg, filter_field):
+    enabled = ["ERROR", "WARNING", "INFO", "DEBUG"]  # Log levels are "DEBUG", "INFO", "WARNING", "ERROR"
+    if level in enabled:  # and filter_field == "0031CDA11AFD98A9":
+        print("[" + str(level) + "] " + str(filter_field) + ": " + str(msg))
+        
+        
+def load_scenario(operator, context, collection, path_to_alocs_json):
```

Ruff (0.14.8)

Unused function argument: `operator`

(ARG001)